src/os/bluestore/BitMapAllocator.cc

```diff
@@ -23,7 +23,7 @@ BitMapAllocator::BitMapAllocator(int64_t device_size, int64_t block_size)
   : m_num_uncommitted(0),
     m_num_committing(0)
 {
-  int64_t zone_size_blks = 1024; // Change it later
+  int64_t zone_size_blks = g_conf->bluestore_bitmapallocator_blocks_per_zone;
```

Comment:
It will be good if you can add some unit test cases for it. At least 512 and 2048 sizes.